src/Contexts/Adaptors/EVMAdaptors.tsx

```diff
@@ -301,19 +300,30 @@ export const EVMHomeAdaptorProvider = ({
       }${decodedRecipient.substr(2)}`; // recipientAddress (?? bytes)
 
       try {
+        const lastBlock = await provider?.getBlock(-1);
+
+        const baseFee = lastBlock?.baseFeePerGas?.mul(125).div(100);
```

Comment:
Can you explain, why multiplying by 125 and div by 100? Is that to always raise 25% the base fee? If so, why and why 25%?

Reply:
So with the new EIP-1559 protocol, the next block's `baseFee` could increase by 12.5% from the previous block. So here I am getting the previous block's `baseFee` and then increasing it by 12.5% which is the maximum possible `baseFee`. I had to do `mul(125).div(100)` since `ethers`' `BigNumber` type does not support decimals.

Reply:
Ok, but you are increasing by 25% right? or my math is off?

Reply:
Your math was on point :facepalm:
Fixed.